#! /bin/sh /usr/share/dpatch/dpatch-run
## 31_usermanual-dash.dpatch by Roland Rosenfeld <roland@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: remove "--" from log comments, because jade doesn't like these.

@DPATCH@
diff -urNad privoxy~/doc/source/user-manual.sgml privoxy/doc/source/user-manual.sgml
--- privoxy~/doc/source/user-manual.sgml
+++ privoxy/doc/source/user-manual.sgml
@@ -8913,11 +8913,11 @@
  Update embedded show-url-info output.
 
  Revision 2.37  2007/08/27 16:09:55  fabiankeil
- Fix --pre-chroot-nslookup description which I failed to
+ Fix -pre-chroot-nslookup description which I failed to
  copy and paste properly. Reported by Stephen Gildea.
 
  Revision 2.36  2007/08/26 16:47:14  fabiankeil
- Add Stephen Gildea's --pre-chroot-nslookup patch [#1276666],
+ Add Stephen Gildea's -pre-chroot-nslookup patch [#1276666],
  extensive comments moved to user manual.
 
  Revision 2.35  2007/08/26 14:59:49  fabiankeil
